app/views/widgets/components/code-editor.vue

```diff
@@ -17,80 +17,117 @@
         
 <script>
 
-    define(["Vue", 'code-editor-vue', 
-        'css!https://cdn.cbd.int/codemirror@5.58.3/theme/base16-dark.css',
-        'https://cdn.cbd.int/codemirror@5.58.3/mode/xml/xml',
-        'https://cdn.cbd.int/codemirror@5.58.3/mode/handlebars/handlebars',
-        'https://cdn.cbd.int/codemirror@5.58.3/mode/htmlmixed/htmlmixed',       
-        'https://cdn.cbd.int/codemirror@5.58.3/mode/javascript/javascript',
-        'https://cdn.cbd.int/codemirror@5.58.3/addon/selection/active-line',
-        'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/closetag',
-        'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/matchbrackets',
-        'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/continuecomment.js',
-        'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/comment.js',
-        'css!https://cdn.cbd.int/codemirror@5.58.3/lib/codemirror.css',
-        'css!https://cdn.cbd.int/codemirror@5.58.3/theme/base16-dark.css'
-    ], function (vue, codemirror) {
+import codemirror from 'code-editor-vue'
+
+import 'https://cdn.cbd.int/codemirror@5.58.3/mode/xml/xml';
+import 'https://cdn.cbd.int/codemirror@5.58.3/mode/handlebars/handlebars';
+import 'https://cdn.cbd.int/codemirror@5.58.3/mode/htmlmixed/htmlmixed';
+import 'https://cdn.cbd.int/codemirror@5.58.3/mode/javascript/javascript';
+import 'https://cdn.cbd.int/codemirror@5.58.3/addon/selection/active-line';
+import 'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/closetag';
+import 'https://cdn.cbd.int/codemirror@5.58.3/addon/edit/matchbrackets';
+import 'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/continuecomment.js';
+import 'https://cdn.cbd.int/codemirror@5.58.3/addon/comment/comment.js';
+import 'css!https://cdn.cbd.int/codemirror@5.58.3/theme/base16-dark.css';
+import 'css!https://cdn.cbd.int/codemirror@5.58.3/lib/codemirror.css';
+import 'css!https://cdn.cbd.int/codemirror@5.58.3/theme/base16-dark.css';
```

Comment:
can this be loded from jsdelievr? using `cdn!`plugin
I think we have to get ride of cbd.cbd .int